Documentation/dev/libvirt-howto.md

```diff
@@ -85,6 +85,35 @@ iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.124.1 --dport 16509 \
   -j ACCEPT -m comment --comment "Allow insecure libvirt clients"
 ```
 
+If using `firewalld`, simply optain the name of the existing active zone which
+can be used to integrate the appropriate source and ports to allow connections from
+the IP range used by your cluster nodes. An example is shown below.
+
+```console
+$ sudo firewall-cmd --get-active-zones
+FedoraWorkstation
+  interfaces: enp0s25 tun0
+```
+With the name of the active zone, include the source and port to allow connections
+from the IP range used by your cluster nodes. The default subnet is `192.168.124.0/24`
+unless otherwise specified.
+
+```sh
+sudo firewall-cmd --zone=FedoraWorkstation --add-source=192.168.124.0/24
+sudo firewall-cmd --zone=FedoraWorkstation --add-port=16509/tcp
+```
+
+Verification of the source and port can be done listing the zone
+
+```sh
+sudo firewall-cmd --zone=FedoraWorkstation --list-ports
+sudo firewall-cmd --zone=FedoraWorkstation --list-sources
+```
+
+NOTE: When the firewall rules are no longer needed, `firewalld --reload`
```

Comment:
Oops, looks like this should have been `firewall-cmd --reload`, based on the example below :/.